### DISCUSSION TITLE
Taskbar Styler: Hide labels when never combining!

### DISCUSSION THREAD

**Discussion by mattcoz** [ramensoftware/windhawk-mods#320] [General]
Windows 11 now has the option to never combine taskbar items, but no option to hide labels when using that option. Taskbar Styler to the rescue! Here's how to do it:

**Target** `taskbar:TaskListLabeledButtonPanel > TextBlock`
**Styles** `Visibility=Collapsed`

If anyone is able to figure out how to show the window title on the thumbnail preview when using this option, please share. I've been unable to target the thumbnails using UWSpy.

**Comment by m417z** [comment]
The thumbnail preview uses native UI, not UWP, so UWPSpy won't help. An alternative solution to what you did is to keep the labels hidden and use the [Disable grouping on the taskbar](https://windhawk.net/mods/taskbar-grouping) mod.

**Comment by mattcoz** [comment]
Ahh, that makes sense, thanks. I tried the disable grouping mod but it had
a few downsides and this is working pretty great. Although, that was a
while ago and now I see that some of those were addressed, so maybe I'll
give it another shot.

On Fri, Sep 29, 2023, 6:14 PM Michael Maltsev ***@***.***>
wrote:

> The thumbnail preview uses native UI, not UWP, so UWPSpy won't help. An
> alternative solution to what you did is to keep the labels hidden and use
> the Disable grouping on the taskbar
> <https://windhawk.net/mods/taskbar-grouping> mod.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ramensoftware/windhawk-mods/discussions/320#discussioncomment-7149019>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACR737MAGE64QAPAFBQO23X45I47ANCNFSM6AAAAAA5LGK35M>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
- Matt
